uid2_client/input_util.py

```diff
@@ -119,3 +120,11 @@ def normalize_and_hash_phone(phone):
     if not is_phone_number_normalized(phone):
         raise ValueError("phone number is not normalized: " + phone)
     return get_base64_encoded_hash(phone)
+
+
+def get_datetime_iso_format(timestamp):
```

Comment:
could you add some unit tests to ensure the timestamp is converted to right iso format?

Reply:
I removed this function as Matt suggested. And I added more test cases in `test_identity_buckets_invalid_timestamp` test